specification/hybridconnectivity/resource-manager/readme.md

```diff
@@ -27,9 +27,17 @@ These are the global settings for the hybridconnectivity.
 ``` yaml
 openapi-type: arm
 openapi-subtype: rpaas
-tag: package-2023-03
+tag: package-2024-12-01
 ```
 
+### Tag: package-2024-12-01
+
+These settings apply only when `--tag=package-2024-12-01` is specified on the command line.
+
+```yaml $(tag) == 'package-2024-12-01'
+input-file:
+  - Microsoft.HybridConnectivity/PublicCloud/stable/2024-12-01/publicCloud.json
+```
 
```

Comment:
Please add another suppression for BodyTopLevelProperties as per this [discussion](https://teams.microsoft.com/l/message/19:0351f5f9404446e4b4fd4eaf2c27448d@thread.skype/1723533919287?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&parentMessageId=1723533919287&teamName=Azure%20SDK&channelName=API%20Spec%20Review&createdTime=1723533919287). It looks like the BodyTopLevelProperties in known issue.